src/consensus/pbft_manager.cpp

```diff
@@ -239,16 +239,20 @@ void PbftManager::setPbftStep(size_t const pbft_step) {
   db_->savePbftMgrField(PbftMgrRoundStep::PbftStep, pbft_step);
   step_ = pbft_step;
 
-  // if (step_ > MAX_STEPS) {
-  //   // Note: We calculate the lambda for a step independently of prior steps
-  //   //       in case missed earlier steps.
-  //   // LAMBDA_ms = 100 * LAMBDA_ms_MIN;
-  //   // LOG(log_nf_) << "Surpassed max steps, relaxing lambda to " << LAMBDA_ms
-  //   //             << " ms in round " << getPbftRound() << ", step " << step_;
-  //   LAMBDA_ms = LAMBDA_ms_MIN;
-  // } else {
-  //   LAMBDA_ms = LAMBDA_ms_MIN;
-  // }
+  if (step_ > MAX_STEPS && LAMBDA_backoff_multiple < 8) {
+    // Note: We calculate the lambda for a step independently of prior steps
+    //       in case missed earlier steps.
+    std::default_random_engine generator;
```

Comment:
this needs to be global or class variable or you need to use some kind of seed (time?) every time you initialize it otherwise will get same number (probably 0) all the time from distribution(0, step_ - MAX_STEPS);

Reply:
@MatusKysel  ok I updated it as you wisely suggested, please re-review.